cmd/code.go

```diff
@@ -1,133 +1,9 @@
 package cmd
 
 import (
-	"errors"
-	"fmt"
-	"os"
-	"os/exec"
-
 	"github.com/spf13/cobra"
-	"github.com/unweave/cli/config"
-	"github.com/unweave/cli/session"
-	"github.com/unweave/cli/ssh"
-	"github.com/unweave/cli/ui"
-	"github.com/unweave/unweave/api/types"
 )
 
 func Code(cmd *cobra.Command, args []string) error {
-	cmd.SilenceUsage = true
-	ctx := cmd.Context()
-
-	execCh := make(chan types.Exec)
-	errCh := make(chan error)
-	isNew := false
-
-	var err error
-	var execRef string // Can be execID or name
-
-	if config.CreateExec {
-
-		ui.Infof("Initializing node...")
-
-		execCh, errCh, err = execCreateAndWatch(ctx, types.ExecConfig{}, types.GitConfig{})
-		if err != nil {
-			return err
-		}
-		isNew = true
-
-	} else {
-		var createNew bool
-
-		if execRef == "" {
-			execRef, createNew, err = sessionSelectSSHExecRef(cmd, execRef, false)
-			if err != nil {
-				return err
-			}
-		}
-
-		if createNew {
-			execCh, errCh, err = execCreateAndWatch(ctx, types.ExecConfig{}, types.GitConfig{})
-			if err != nil {
-				return err
-			}
-			isNew = true
-		} else {
-			execCh, errCh, err = session.Wait(ctx, execRef)
-			if err != nil {
-				return err
-			}
-		}
-	}
```

Comment:
This is removed in favor of the SessionConnect method which removes redundancies between the SSH and Code methods.